webapp/cypress.json

```diff
@@ -3,7 +3,8 @@
     "baseUrl": "http://localhost:8088",
     "testFiles": [
         "**/login*.ts",
-        "**/create*.ts"
+        "**/create*.ts",
+        "**/manage*.ts"
```

Comment:
This seems somewhat weird to me. Why not suffix all cypress integration files with `.spec.ts` so we could do `"**/*.spec.ts"` here instead?

Reply:
I agree. It's done this way to maintain order but we'll change it next time for a better approach.